### ISSUE TITLE
Diferent Rotation per level

### ISSUE THREAD

**Issue by marcodlima** [blacklabel/grouped_categories#112]
Hi,

Is there a way to make diferent rotations per categories level?

Ex. first level horizontal and second level vertical

Best Regards

Marco Lima

**Comment by stpoa** [comment]
Hi,

Currently there is no such functionality. You could try to wrap code from https://github.com/blacklabel/grouped_categories/blob/master/grouped-categories.js in order to get options for each level separately.

Regards.

**Comment by walreyes** [comment]
Hey @marcodlima did you figured out how to do have the second level vertical?

**Comment by marcodlima** [comment]
@walreyes yes i did! I had to add some code to allow that option when setting the categories.

**Comment by drelei** [comment]
@marcodlima, could you suggest where you added the code to have different rotation levels? Thank you.

**Comment by tomassatinsky** [comment]
@drelei use the `groupedOptions` property to set properties by level
